Subject: [PATCH] Belle's meat 1 code
---
Index: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Elevator.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Elevator.java b/TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Elevator.java
--- a/TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Elevator.java	(revision 44cb9180b36645385d9dc572196d783e3be034d4)
+++ b/TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Elevator.java	(revision 68caf098292b3bda87c4152e804bb2b0b172daff)
@@ -14,8 +14,9 @@
    private DcMotor elevator_Motor;
    private final double raiseSpeed = 1.0;
    private final double lowerSpeed = -0.7;
-   private final int bottomBucketPosition = 750;
-   private final int topBucketPosition = 1300;
+   private final int wallElementPosition = 240;
+   private final int lowRungPosition = 440;
+   private final int highRungPosition = 1517;
 
    // Motor PIDF coefficients, USE CAUTION. These values change how the motor
    // responds when commanded to an encoder position.
@@ -86,29 +87,42 @@
       }
    }
 
-   public void toBottomBucket()
+   public void toWallElementPosition()
    {
-      if (this.getPosition() < bottomBucketPosition) {
+      if (this.getPosition() < wallElementPosition) {
          elevator_Motor.setPower(raiseSpeed);
       }
       else {
          elevator_Motor.setPower(lowerSpeed);
       }
 
-      elevator_Motor.setTargetPosition(bottomBucketPosition);
+      elevator_Motor.setTargetPosition(wallElementPosition);
+      elevator_Motor.setMode(DcMotor.RunMode.RUN_TO_POSITION);
+   }
+
+   public void toLowRungPosition()
+   {
+      if (this.getPosition() < lowRungPosition) {
+         elevator_Motor.setPower(this.raiseSpeed);
+      }
+      else {
+         elevator_Motor.setPower(this.lowerSpeed);
+      }
+
+      elevator_Motor.setTargetPosition(this.lowRungPosition);
       elevator_Motor.setMode(DcMotor.RunMode.RUN_TO_POSITION);
    }
 
-   public void toTopBucket()
+   public void toHighRungPosition()
    {
-      if (this.getPosition() < topBucketPosition) {
+      if (this.getPosition() < highRungPosition) {
          elevator_Motor.setPower(this.raiseSpeed);
       }
       else {
          elevator_Motor.setPower(this.lowerSpeed);
       }
 
-      elevator_Motor.setTargetPosition(this.topBucketPosition);
+      elevator_Motor.setTargetPosition(highRungPosition);
       elevator_Motor.setMode(DcMotor.RunMode.RUN_TO_POSITION);
    }
 
Index: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Gripper.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Gripper.java b/TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Gripper.java
--- a/TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Gripper.java	(revision 44cb9180b36645385d9dc572196d783e3be034d4)
+++ b/TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Gripper.java	(revision 68caf098292b3bda87c4152e804bb2b0b172daff)
@@ -7,8 +7,8 @@
 {
    public Servo gripper_servo;
    private String state;
-   private final double openPosition = 0.3;
-   private final double closedPosition = 0.7;
+   private final double openPosition = 0.35;
+   private final double closedPosition = 0.8;
 
    public void init(HardwareMap hwMap, Blinkin blinkin)
    {
Index: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/RobotManual.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/TeamCode/src/main/java/org/firstinspires/ftc/teamcode/RobotManual.java b/TeamCode/src/main/java/org/firstinspires/ftc/teamcode/RobotManual.java
--- a/TeamCode/src/main/java/org/firstinspires/ftc/teamcode/RobotManual.java	(revision 44cb9180b36645385d9dc572196d783e3be034d4)
+++ b/TeamCode/src/main/java/org/firstinspires/ftc/teamcode/RobotManual.java	(revision 68caf098292b3bda87c4152e804bb2b0b172daff)
@@ -38,10 +38,13 @@
 
       // --- Encoder elevator control ------------------
       if (gamepad2.triangle) {
-         elevator.toTopBucket();
+         elevator.toLowRungPosition();
       }
       else if (gamepad2.circle) {
-         elevator.toBottomBucket();
+         elevator.toWallElementPosition();
+      }
+      else if (gamepad2.square) {
+         elevator.toHighRungPosition();
       }
       else if (gamepad2.cross) {
          elevator.toHome();
